src/video/video_driver.hpp

```diff
@@ -222,6 +222,9 @@ class VideoDriver : public Driver {
 		bool unlock; ///< Stores if the lock did anything that has to be undone.
 	};
 
+	bool isFastForwardViaKey(){
+		return fast_forward_via_key;
+	}
```

Comment:
```suggestion
	bool IsFastForwardViaKey()
	{
		return this->fast_forward_via_key;
	}
	
```
Applying https://wiki.openttd.org/en/Development/Coding%20style